modules/api/src/main/java/org/apache/ignite/table/Tuple.java

```diff
@@ -35,6 +35,8 @@
  * Column name arguments passed to the tuple methods must use SQL-parser style notation; e.g.,
  * "myColumn" - column named "MYCOLUMN", "\"MyColumn\"" - "MyColumn", etc.
  *
+ * <p>Note: case sensitive name "\"MYCOLUMN\"" is equivalent to case sensitive name "MYCOLUMN".
```

Comment:
I think it's not quite correct to mentioning "case sensitivity" In this context. In our system case sensitive comparison is used for dealing with identifiers, user cannot change this or specify some "case insensitive" names for certain columns. What user can change though is whether the name will be _normalized_ or not. During normalization, all letters in lower case will be replaced with similar letters in upper case. To omit normalization (as well as disable validation of characters used), user can wrap name with double quotes.